cardano-testnet/src/Testnet/Options.hs

```diff
@@ -30,3 +42,112 @@ defaultTestnetOptions = BabbageTestnetOptions
   , babbageTotalBalance = 10020000000
   , babbageNodeLoggingFormat = NodeLoggingFormatAsJson
   }
+
+
+defaultYamlConfig :: KeyMapAeson.KeyMap Aeson.Value
+defaultYamlConfig =
+  mconcat $ map (uncurry KeyMapAeson.singleton)
+    [ ("ApplicationName", "cardano-sl")
+    , ("ApplicationVersion", Aeson.Number 1)
+    , ("Protocol", "Cardano")
+    , ("SocketPath", "db/node.socket")
+    , ("PBftSignatureThreshold", Aeson.Number (fromFloatDigits (0.6 :: Double)))
+    , ("minSeverity", "Debug")
+    , ("EnableLogMetrics", Aeson.Bool False)
+    , ("TurnOnLogMetrics", Aeson.Bool False)
+    , ("MaxConcurrencyBulkSync", Aeson.Number 1)
+    , ("MaxConcurrencyDeadline", Aeson.Number 2)
+    , ("EnableLogging", Aeson.Bool True)
+    , ("ByronGenesisFile", "byron/genesis.json")
+    , ("ShelleyGenesisFile", "shelley/genesis.json")
+    , ("AlonzoGenesisFile", "shelley/genesis.alonzo.json")
+    , ("ConwayGenesisFile", "shelley/genesis.conway.json")
+    , ("RequiresNetworkMagic", "RequiresMagic")
+    , ("LastKnownBlockVersion-Major", Aeson.Number 6)
+    , ("LastKnownBlockVersion-Minor", Aeson.Number 0)
+    , ("LastKnownBlockVersion-Alt", Aeson.Number 0)
+    , ("TraceBlockchainTime", Aeson.Bool True)
+    , ("ExperimentalHardForksEnabled", Aeson.Bool True)
+    , ("TestShelleyHardForkAtEpoch", Aeson.Number 0)
+    , ("TestAllegraHardForkAtEpoch", Aeson.Number 0)
+    , ("TestMaryHardForkAtEpoch", Aeson.Number 0)
+    , ("TestAlonzoHardForkAtEpoch", Aeson.Number 0)
+    , ("TestEnableDevelopmentHardForkEras", Aeson.Bool True)
+    , ("EnableP2P", Aeson.Bool False)
+    , ("setupScribes", setupScribes)
+    , ("rotation", rotationObject)
+    , ("defaultScribes", defaultScribes)
+    , ("setupBackends", Aeson.Array $ Vector.fromList ["KatipBK"])
+    , ("defaultBackends", Aeson.Array $ Vector.fromList ["KatipBK"])
+    , ("options", Aeson.object mempty)
```

Comment:
It might be useful to have comments here about where these values come from or why these particular values were chosen if that information is available.